mlflow/R/mlflow/R/model.R

```diff
@@ -47,6 +47,32 @@ mlflow_log_model <- function(model, artifact_path, ...) {
   res
 }
 
+#' Write MLflow model spec
+#'
+#' Writes an `MLmodel` file into the directory specified by `path`. The `content` argument is a list
```

Comment:
Something to consider when continuing work on this is how to support signature functionality. 
Could you share your thoughts on how to do this enforcement on data structures and what data types you are proposing to (and not to) support? 

Another consideration is to support inference parameters (for Python, the implementation supports the concept of a dictionary of parameter keys and default values that is included with a model signature at save / log time). 

It would be very beneficial to include support for this functionality for R flavors in general, but particularly for the custom models.

Reply:
Sorry for the slow response, I'm partially off-grid at the moment. R has great support for code introspection so generating a signature should be straightforward 🤞 . 

How do we get the signature into MLFlow? Should it be part of the model spec, and if so, please could you point me towards the source for the Python version so I can copy? The question of what types of model to support is a bigger discussion so I'll pick that up in the main thread.